PortalSwift/Sources/PortalSwift/Storage/PasswordStorage.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  PasswordStorage.swift
+//
+//  Created by Portal Labs, Inc.
+//  Copyright © 2022 Portal Labs, Inc. All rights reserved.
+//
+
+import Foundation
+
+public enum PasswordStorageError: Error {
+  case passwordMissing(String)
+  case unableToEncodeData
+}
+
+/// Responsible for CRUD actions for items in the specified storage.
+public class PasswordStorage: Storage, PortalStorage {
+  public var api: PortalApi?
+  public let encryption: PortalEncryption
```

Comment:
need to double check that this handles password correctly since its a different encrypt and decrypt